kubectl-apply-all.sh

```diff
@@ -12,4 +12,10 @@
 #
 # Apply the base Soucegraph deployment
 # shellcheck disable=SC2068
+
+# Run the DB migration job and wait for it to complete
+kubectl delete -f base/migrator/migrator.Job.yaml --ignore-not-found=true
+kubectl apply -f base/migrator/migrator.Job.yaml
+kubectl wait -f base/migrator/migrator.job.yaml --for=condition=complete
```

Comment:
How does this command behave when the migration job fails, or takes longer than the default timeout (30s)?